src/components/Homepage/CurrentEvents.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import config from "../../_config";
+import "./Homepage.scss";
+import Fade from "react-reveal/Fade";
+import EventCard from "../Opportunities/EventCard"
+
+function CurrentEvents(props) {
+  const [events, setEvents] = useState([]);
+
+  useEffect(() => {
+    fetch(config.url + "/events", {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        setEvents(Object.keys(data).map((uid) => data[uid]));
+      })
+      .catch((error) => {
+        setEvents([]);
+        console.error("Error:", error);
```

Comment:
Remove or wrap in a check for "developer mode" toggle. We don't want this error displayed in production.
```suggestion
```